README.md

```diff
@@ -52,11 +52,15 @@ fastify.register(mail, pov: { {engine: { TEMPLATE_ENGINE_OBJECT } }, transporter
 
 // setup test route
 fastify.get("/sendmail", async (req, reply) => {
-  const recipients = ["test@example.com"]
-  const templates = "path/to/my/templates/"
+  const templatePath = "path/to/my/templates/"
   const context = { name: "Test Name" }
 
-  const queued = await fastify.mail.sendMail(recipients, templates, context)
+  const opts = {
```

Comment:
as we're using one liners, could simplify for readability down to:

```
const opts = {
  templatePath: "path/to/my/templates/",
  context: { name: "Test Name" }
}
```

Reply:
done